docs/ModelStore.md

```diff
@@ -0,0 +1,54 @@
+# Model Store
+
+
+## Introduction
+Model Store is a new feature in DIGITS.
+It lists models in user-specified servers and imports them into DIGITS.
+
+
+## Setting up environment variable
+The configuration of Model Store requires one environment variable DIGITS_MODEL_STORE_URL to be set.
+NVIDIA plans to publish one public Model Store at http://developer.download.nvidia.com/compute/machine-learning/modelstore/4.5.0.
+You can set up the environment variable with that url before launching DIGITS.
+For example, run the following command in your Bash shell.
+``` shell
+export DIGITS_MODEL_STORE_URL='http://developer.download.nvidia.com/compute/machine-learning/modelstore/4.5.0'
+```
+If multiple model stores are available, specify their url's, separated by the comma (,).
+``` shell
+export DIGITS_MODEL_STORE_URL='http://localhost/mymodelstore,http://dlserver/teammodelstore'
+```
+
+
+## Browse and import model
+First launch DIGITS.
+Click Pretrained Models tab, and select 'Retrieve from Model Store' under 'Load Model.'
+The new page shows models available in model stores.
```

Comment:
I also think screenshots would be much better than sentences after sentences (for most readers like me).  I will add some.